pkg/sql/schema_resolver.go

```diff
@@ -69,18 +70,27 @@ type schemaResolver struct {
 // GetObjectNamesAndIDs implements the resolver.SchemaResolver interface.
 func (sr *schemaResolver) GetObjectNamesAndIDs(
```

Comment:
I feel like we've got a terminology problem these days: is a function an object? 

By my old definition, it is, but it isn't here.

Reply:
It is, but it's also a special kind of object in that unlike other objects we don't expect functions to have a namespace entry.

I completely agree with you in general, btw. Untangling `GetObjectNamesAndIDs` is going to be its own unit of work focused on the resolver interface, and it's something I want to do this week.